bot.user.js

```diff
@@ -579,7 +579,8 @@ window.computeFoodGoal = function() {
 			for (var j = 0; j < nIter; ++j) {
 				var p2 = window.sortedFood[j];
 				var dist = window.getDistance(p1.xx, p1.yy, p2.xx, p2.yy);
-				if (dist < 100) {
+				
+				if (dist < getSnakeWidth() * 5) {
```

Comment:
Good idea

On Saturday, May 7, 2016, Chad Z <notifications@github.com> wrote:

> In bot.user.js
> <https://github.com/ErmiyaEskandary/Slither.io-bot/pull/95#discussion_r62424857>
> :
>
> > @@ -579,7 +579,8 @@ window.computeFoodGoal = function() {
> >  			for (var j = 0; j < nIter; ++j) {
> >  				var p2 = window.sortedFood[j];
> >  				var dist = window.getDistance(p1.xx, p1.yy, p2.xx, p2.yy);
> > -				if (dist < 100) {
> > +				
> > +				if (dist < getSnakeWidth() * 5) {
>
> Too big. it should be aiming at clusters it can completely eat, no? (i.e.
> one snake width)
>
> I'm going to try both 5x and 1x to see what works better.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ErmiyaEskandary/Slither.io-bot/pull/95/files/298b5406e2d65659bda97028a6a4793cf9357c80..5633bab8ce17c7eb5aa6de7739d1b65cb7d871a9#r62424857>
>